.github/workflows/image-push-release.yml

```diff
@@ -0,0 +1,41 @@
+name: "Publish release image"
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    env:
+      IMAGE_NAME: ghcr.io/${{ github.repository }}
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to Docker
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ${{ env.IMAGE_NAME }}
+          tags: type=semver,pattern={{version}}
+          flavor: latest=false
+      - name: Build and push accelerated-bridge-cni
+        uses: docker/build-push-action@v2
```

Comment:
can this be merged with image-push-master into `image-push.yaml` ?

using `if` expression and triggering the job on push to master and tag ?

Reply:
https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions

Reply:
ok. will do

Reply:
Done. Updated code in PR #27